__tests__/api/disburse_loan.js

```diff
@@ -0,0 +1,37 @@
+const frisby = require ('../../test_config.js');
+const Joi = frisby.Joi;
+
+it ('POST should return a status of 200 OK', function (res) {
+  return frisby
+    .put('http://localhost:3000/loans/disburse/5b3c114131124a0d7415c640', {
+      amountDisbursed: 3500000 ,
```

Comment:
Are you posting amount disbursed and error message while disbursing the loan. If yes, Why ?

Reply:
The amount disbursed is to be posted by the admin. Peradventure, the total amount of loan requested cannot be disbursed a note is attached to disclose why the entire loan cannot be disbursed. The note will be null if all the amount is been disbursed

Reply:
It is not an error message. It is a short note attached to explain why all the loan cannot be disbursed